assets/translations/en.json

```diff
@@ -47,5 +47,7 @@
   "enterAmountDescription": "Enter the amount you want to fund your wallet and create wagers.",
   "fund": "Fund",
   "addHashtag(s)": "Add Hashtag(s)",
-  "Hashtagshelpotherusersfindyourwagereasilyandquickly": "Hashtags help other users find your\nwager easily and quickly"
-}
\ No newline at end of file
+  "Hashtagshelpotherusersfindyourwagereasilyandquickly": "Hashtags help other users find your\nwager easily and quickly",
+  "recentTransactions": "Recent Transactions",
+  "walletCaps": "WALLET"
```

Comment:
we can't have this. instead, under the extensions folder. create a string extension that takes in a string and return the same string in Uppercase. then use appropriately